src/Home.js

```diff
@@ -87,119 +99,165 @@ export default class Home extends Component {
-              <Anchor href="http://slackin.grommet.io"
+          <Footer appCentered={true} justify='center'>
+            <Menu inline={true} responsive={false} direction='row'>
+              <Anchor href='http://slackin.grommet.io'
                 icon={<SlackIcon a11yTitle='Grommet Slack' />}/>
-              <Anchor href="https://github.com/grommet/grommet"
+              <Anchor href='https://github.com/grommet/grommet'
                 icon={<GithubIcon a11yTitle='Grommet Github' />}/>
-              <Anchor href="https://twitter.com/grommetux"
-                icon={<TwitterIcon colorIndex="grey-4"
+              <Anchor href='https://twitter.com/grommetux'
+                icon={<TwitterIcon colorIndex='grey-4'
                 a11yTitle='Grommet Twitter' />}/>
             </Menu>
           </Footer>
         </HomeSection>
 
-        <HomeSection ref="mobile"
-          backgroundImage={'url(/img/mobile_first.jpg)'} justify="start"
-          colorIndex="dark" style={mobileStyle}>
-          <Box className="home__mobile" align="center">
-            <Heading tag="h2" align="center">
+        <HomeSection ref={(ref) => this._mobileRef = ref}
+          backgroundImage={'url(/img/mobile_first.jpg)'} justify='start'
+          colorIndex='dark' style={mobileStyle}>
+          <Box className='home__mobile' align='center'>
+            <Heading tag='h2' align='center'>
               Mobile-first ready for business
             </Heading>
-            <Paragraph align="center">Think small. Starting with mobile-first
+            <Paragraph align='center'>Think small. Starting with mobile-first
               design allows for easy app scalability to larger
               devices.</Paragraph>
             {/*}
-            <NavButton path="/docs/learn" label="Learn" />
+            <NavButton path='/docs/learn' label='Learn' />
             {*/}
           </Box>
         </HomeSection>
 
+        <HomeSection colorIndex='brand'>
+          <Heading tag='h2'>Why Grommet?</Heading>
+          <Box direction='row' wrap={true}
+            pad={{
+              between: 'large', vertical: 'medium', horizontal: 'medium'
+            }}>
+            <WhyGrommetItem icon={<ResourcesIcon size='xlarge' />}
+              heading='Comprehensive'>
+              <Paragraph align='center'>
+                A curated design and development platform that is more than
+                just a Facebook React widget library. Grommet offers the tools
+                to create amazing experiences.
+              </Paragraph>
+            </WhyGrommetItem>
+            <WhyGrommetItem icon={<ConfigureIcon size='xlarge' />}
+              heading='Flexible'>
+              <Paragraph align='center'>
+                Responsive web development without the grid management,
+                theming that allows your brand to shine, and design
+                componentry that is loaded on-demand.
```

Comment:
"Responsive web without the hassle of grid management and modular components loaded on-demand.  Themes allow your brand to shine."